<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfilSayfasi</class>
 <widget class="QDialog" name="ProfilSayfasi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profil</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFE0B2;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="baslikLabel">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 150);
border: 1px solid rgba(0, 0, 0, 0);
border-bottom: 2px solid #FFA000;
color: #5D4037;
padding-bottom: 7px;
border-radius: 4px;</string>
     </property>
     <property name="text">
      <string>Kullanıcı Profili</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="kullaniciAdiLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Kullanıcı Adı:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="kullaniciAdiLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
border: 1px solid #ced4da;
border-radius: 4px;
padding: 6px;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="sifreLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Şifre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="sifreLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
border: 1px solid #ced4da;
border-radius: 4px;
padding: 6px;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="epostaLabel">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>E-posta:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="epostaLineEdit">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: white;
border: 1px solid #ced4da;
border-radius: 4px;
padding: 6px;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="guncelleButon">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #FF9800;
    color: black;
    border-radius: 15px;
    padding: 8px 16px;
    font-weight: bold;
    border: 2px solid #FF9800;
}

QPushButton:hover {
    background-color: #FB8C00;
    border: 2px solid #FB8C00;
}

QPushButton:pressed {
    background-color: #F57C00;
    border: 2px solid #F57C00;
}</string>
     </property>
     <property name="text">
      <string>Güncelle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gecmisGroupBox">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">margin-top: 20px;
border: 1px solid #e0e0e0;
border-radius: 5px;
padding: 10px;</string>
     </property>
     <property name="title">
      <string>Geçmiş</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="gecmisTextEdit">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 200);
border: 1px solid #FFA726;
border-radius: 5px;
padding: 5px;
</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Burada geçmiş işlemleriniz listelenecektir.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
